ajax/libs/jquery.form/package.json

```diff
@@ -2,41 +2,23 @@
   "name": "jquery.form",
   "filename": "jquery.form.min.js",
   "title": "jQuery Form Plugin",
-  "description": "A simple way to AJAX-ify any form on your page; with file upload and progress support.",
-  "keywords": [
-    "form",
-    "upload",
-    "ajax"
-  ],
-  "version": "3.51",
-  "author": {
```

Comment:
Could you please tell why to remove this field? Is the informatiom wrong? Could you provide the reference? Thank you!